INFO     Logger:page.py:133 Page.wait_until_stable() - Page wait until driver stable
INFO     Logger:page.py:243 Page.set_page_load_timeout() - Set page load timeout: '30'
INFO     Logger:page.py:227 Page.maximize_window() - Maximize browser window
INFO     Logger:page.py:59 User Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
INFO     Logger:page.py:112 Page.visit() - Get URL: 'https://next.4zida.rs'
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//h2'
INFO     Logger:page.py:115 Error: Application error: a client-side exception has occurred (see the browser console for more information).
INFO     Logger:page.py:112 Page.visit() - Get URL: 'https://next.4zida.rs'
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//h2'
INFO     Logger:page.py:118 Client window width: 393, window height: 873
INFO     Logger:page.py:118 Client window width: 393, window height: 873
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//header/button'
INFO     Logger:element.py:90 Element.scroll_to_element() - Scroll to this element
INFO     Logger:element_should.py:59 Element is visible
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//header/button'
INFO     Logger:element.py:90 Element.scroll_to_element() - Scroll to this element
INFO     Logger:element.py:38 Element.click() - Click this element
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//header//nav//div[text()="Prodaja"]'
INFO     Logger:element.py:90 Element.scroll_to_element() - Scroll to this element
INFO     Logger:element_should.py:59 Element is visible
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//header//nav//div[text()="Prodaja"]'
INFO     Logger:element.py:90 Element.scroll_to_element() - Scroll to this element
INFO     Logger:element.py:38 Element.click() - Click this element
INFO     Logger:page.py:149 Page.get_xpath() - Get the element with xpath: '//header//nav/a[text()="Vodič za kupovinu"]'
INFO     Logger:element.py:90 Element.scroll_to_element() - Scroll to this element
INFO     Logger:element_should.py:35 Element is displayed and enabled (clickable)
INFO     Logger:element.py:38 Element.click() - Click this element
ERROR    Logger:page.py:322 Expected URL: 'https://next.4zida.rs/blog/vodic-za-kupovinu-nekretnine' - Actual URL: 'https://www.4zida.rs/blog/vodic-za-kupovinu-nekretnine/'
INFO     Logger:http_requests.py:12 GET 'https://www.4zida.rs/blog/vodic-za-kupovinu-nekretnine/'. Response status code: 200
INFO     Logger:page.py:220 Page.screenshot() - Save screenshot to: 'screenshots/test_navigation_menu_sale[20]2024-03-05T180435.png'
INFO     Logger:conftest.py:55 Browser console logs: [{'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/chunks/app/(base)/layout-913d2c48fcd1a1a8.js - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864557}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/media/88325a2c1fede2f4-s.woff2 - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864563}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/image?url=https%3A%2F%2Fresizer2.4zida.rs%2Funsigned%2Frs%3Afit%3A125%3A125%2Fplain%2Flocal%3A%2F%2F%2Fagencies%2F373c3b631c.jpeg%40webp&w=128&q=75 - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864574}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/chunks/719-5ba1782a01cdb373.js - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864575}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/image?url=https%3A%2F%2Fresizer2.4zida.rs%2Funsigned%2Frs%3Afit%3A125%3A125%2Fplain%2Flocal%3A%2F%2F%2Fagencies%2F373e537120.jpeg%40webp&w=128&q=75 - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864576}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/chunks/69-d459d3b3d3497d9e.js 0:3770 ChunkLoadError: Loading chunk 605 failed.\n(error: https://next.4zida.rs/_next/static/chunks/app/(base)/layout-913d2c48fcd1a1a8.js)\n    at s.f.j (https://next.4zida.rs/_next/static/chunks/webpack-1a63d106165023e3.js:1:3169)\n    at https://next.4zida.rs/_next/static/chunks/webpack-1a63d106165023e3.js:1:1385\n    at Array.reduce (<anonymous>)\n    at s.e (https://next.4zida.rs/_next/static/chunks/webpack-1a63d106165023e3.js:1:1351)\n    at https://next.4zida.rs/_next/static/chunks/69-d459d3b3d3497d9e.js:1:84190\n    at https://next.4zida.rs/_next/static/chunks/69-d459d3b3d3497d9e.js:1:84392\n    at t (https://next.4zida.rs/_next/static/chunks/69-d459d3b3d3497d9e.js:1:84595)', 'source': 'console-api', 'timestamp': 1709661864589}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/chunks/fd9d1056-ec99789af646e57c.js 0:117693 Uncaught Error: Minified React error #423; visit https://reactjs.o… for full errors and additional helpful warnings.', 'source': 'javascript', 'timestamp': 1709661864589}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/image?url=https%3A%2F%2Fresizer2.4zida.rs%2Funsigned%2Frs%3Afit%3A125%3A125%2Fplain%2Flocal%3A%2F%2F%2Fagencies%2Fd14fca2f83.png%40webp&w=128&q=75 - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864589}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/_next/static/media/d83fe381bb17eb77-s.woff2 - Failed to load resource: the server responded with a status of 429 ()', 'source': 'network', 'timestamp': 1709661864707}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/investitori?_rsc=tnqvz - Failed to load resource: the server responded with a status of 404 ()', 'source': 'network', 'timestamp': 1709661868138}, {'level': 'SEVERE', 'message': 'https://next.4zida.rs/agencije?_rsc=tnqvz - Failed to load resource: the server responded with a status of 404 ()', 'source': 'network', 'timestamp': 1709661868139}]
INFO     Logger:page.py:212 Page.quit() - Quit page and close all windows from the browser session